.gitignore

```diff
@@ -1,12 +1,12 @@
-.idea/
-*/kafka-streams-examples
-*/delta_configs/
-*/target
-*/*/target
+**/.idea/
+**/kafka-streams-examples
```

Comment:
What's the difference between single and double star?

Reply:
maybe with `kafka-streams-examples` this isn't proper, not sure. With comman output directories like `target/` I wanted to ignore all `target/` folders regardless of location in the tree.

https://www.atlassian.com/git/tutorials/saving-changes/gitignore

Reply:
Gotch, thanks!